packages/cx-layout/src/components/Layout/tokens/cxLayout.ts

```diff
@@ -53,15 +47,27 @@ const Base = asLayoutToken({
   }
 });
 
+/*
+ * Tailwind's container is specifially not used due to its feature it set's max-width
+ * to min-width of breakpoint.  So instead rely on ContainerWrapper to margin percent
+ * to contain content until we get to xl and then constrain by max-width.
+ */
+const ConstrainSite = asLayoutToken({
+  Theme: {
```

Comment:
Seems like these should be in spacing or layout not theme, but also see below... not sure we need this token at all.